src/hydra/dataProvider.js

```diff
@@ -451,6 +451,10 @@ export default (
 
       case UPDATE:
         const updateHttpMethod = extraInformation.hasFileField ? 'POST' : 'PUT';
+
+        // fixes https://github.com/api-platform/admin/issues/259
+        params.data.id = params.originId;
```

Comment:
I think it can be dangerous to modify the params by reference. Maybe it would be better to clone it, don't you think?

Reply:
@alanpoulain you're right, I changed it to copy the values.